gk/main.c

```diff
@@ -2403,6 +2403,7 @@ gk_stage1(void *arg)
 	for (i = 0; i < gk_conf->num_lcores; i++) {
 		unsigned int lcore = gk_conf->lcores[i];
 		struct gk_instance *inst_ptr = &gk_conf->instances[i];
+		struct sol_config *sol_conf = gk_conf->sol_conf;
```

Comment:
Style: although variable `sol_conf` is only used inside of this `for` loop, its value doesn't change. So define it within the outside scope (i.e. at the beginning of `gk_stage1()`), and assign its value just before the `for` loop.